js_modules/dagit/packages/core/src/runs/RunActionsMenu.tsx

```diff
@@ -303,30 +313,32 @@ export const RunBulkActionsMenu: React.FC<{
       <Popover
         content={
           <Menu>
-            {canTerminatePipelineExecution.enabled ? (
+            {canTerminateAny ? (
```

Comment:
Not strictly related to this PR, but I'd be in favor of showing these menu items grayed out with a tooltip rather than hiding them if it's possible -- just so people know that they can / should ask for permission